include/sway/tree/view.h

```diff
@@ -192,8 +192,11 @@ struct sway_view_child_impl {
  */
 struct sway_view_child {
 	const struct sway_view_child_impl *impl;
+	struct wl_list parent_link;
```

Comment:
Can you change this to just `link` for consistency with other struct definitions?

Reply:
Sure. I had taken the naming from wlr_subsurface, but I see now that it was an outlier. Fixed.

Reply:
Yeah, `wlr_subsurface` is very old, so it's not as consistent as other wlroots types. We should probably fix it at some point.